packages/@tinacms/schema-tools/src/types/SchemaTypes.ts

```diff
@@ -98,7 +98,7 @@ interface BaseCollection {
     router?: (args: {
       document: Document
       collection: TinaCloudCollection<true>
-    }) => string
+    }) => string | undefined
```

Comment:
I had thought this could be more strict since it's no longer a global config but you're probably right, doesn't need to enforce this.

Reply:
So undefined just brings the user to the admin instead of a contextual page right? I think your use-case makes sense @logan-anderson. We might just want to add a note to our reference docs if it's not already there.

Reply:
@jamespohalloran Yeah exactly. It should be in the docs PR https://github.com/tinacms/tina.io/pull/1394